index.js

```diff
@@ -48,6 +49,10 @@ export const authorize = ({
   ];
   if (Platform.OS === 'android') {
     nativeMethodArguments.push(dangerouslyAllowInsecureHttpRequests);
+  } else {
+    // add a new useNonce param on iOS to support making it optional
+    const nonceParamIndex = 5;
```

Comment:
This seems a bit fiddly, I'd prefer to just add it to the end as the android-only method above
```js
nativeMethodArguments.push(useNonce);
```